tee-worker/bitacross/enclave-runtime/src/initialization/global_components.rs

```diff
@@ -68,7 +68,7 @@ use itp_sgx_crypto::{
 	ecdsa::{Pair as EcdsaPair, Seal as EcdsaSeal},
 	key_repository::KeyRepository,
 	schnorr::{Pair as SchnorrPair, Seal as SchnorrSeal},
-	Aes, AesSeal, Ed25519Seal, Rsa3072Seal,
+	Aes, AesSeal, Ed25519Seal, IDGraphKey, IDGraphKeySeal, Rsa3072Seal,
```

Comment:
Do these types exist?

Reply:
No - that's why compilation failed : ) I fixed it locally too